### PR TITLE
Fine tune `README.Rmd` regarding validation scope

### PR DESCRIPTION
**Pull request by AARON-CLARK**
Really small PR fine tuning the wording of the README to accommodate [suggestion ](https://github.com/pharmaverse/pharmaverse/issues/195#issuecomment-1498764813)from pharmaverse maintainers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,10 +52,14 @@ reviewer to:
 ### Echo-ing `{riskmetric}`’s Approach to Validation <a href='https://pharmar.github.io/riskmetric/'><img src="man/figures/hex-riskmetric-aspconfig.png" align="right" height="172" style="float:right; height:172px;"/></a>
 
 Validation can serve as an umbrella for various terms, and admittedly,
-companies will diverge on what is the correct approach to validation.
-The validation approach we followed is based on the philosophy of the
-white paper set forth by the R Validation Hub: [White
-Paper](https://www.pharmar.org/white-paper/).
+companies will diverge on what may be the “correct approach”. The
+`riskassessment` app is built on a `rismetric`-foundation, whose
+deverlopers follow the validation philosophy proposed in [this white
```
Comment (rossfarrugia) [inline README.md:57]:
```suggestion
developers follow the validation philosophy proposed in [this white
```

File: README.md [hunk 2]
```diff
@@ -52,10 +52,14 @@ reviewer to:
 ### Echo-ing `{riskmetric}`’s Approach to Validation <a href='https://pharmar.github.io/riskmetric/'><img src="man/figures/hex-riskmetric-aspconfig.png" align="right" height="172" style="float:right; height:172px;"/></a>
 
 Validation can serve as an umbrella for various terms, and admittedly,
-companies will diverge on what is the correct approach to validation.
-The validation approach we followed is based on the philosophy of the
-white paper set forth by the R Validation Hub: [White
-Paper](https://www.pharmar.org/white-paper/).
+companies will diverge on what may be the “correct approach”. The
+`riskassessment` app is built on a `rismetric`-foundation, whose
```
Comment (rossfarrugia) [inline README.md:56]:
```suggestion
`riskassessment` app is built on a `riskmetric`-foundation, whose
```

**Review by rossfarrugia — COMMENTED** [review]
@AARON-CLARK spotted some minor typos

**Comment by AARON-CLARK** [comment]
Hey @Xyarz, this one has been sitting for ~ two biz days. Could you give it a look? Giving Andrew a break since he looked at the last 3 PRs. Thanks!

**Review by Xyarz — APPROVED** [review]

**Pull request merged**